github/github.go

```diff
@@ -739,7 +739,7 @@ func (c *Client) RateLimit() (*Rate, *Response, error) {
 		return nil, nil, err
 	}
 
-	return limits.Core, resp, err
+	return limits.Core, resp, nil
```

Comment:
Well spotted. I agree with checking both limit and error in this case (better than returning nil, nil, nil)

Incidentally, no tests failed. Is there any motivation to increase the test coverage in the project in the future?